README.md

```diff
@@ -1,11 +1,28 @@
 # Driver Drowsiness Detection System
 
 ## Description
+<<<<<<< HEAD
+This is a drowsiness detection system.In the program, we check if the eyes are open or closed for a certain period of time. If closed, the program will alert the user playng an alarm sound.
+
+##Applications
+This can be used to determine if a driver is drunk
+
+##Requirements
+nose
+tensorflow
+opencv-python
+
+##Working Example
+(Add Pictures and Gifs Here)
+
+
+=======
 TODO:
 
 
 ## Comparing Python and Matlab for drowsiness detection
 TODO: Do a comparative study of the 2 libraries and propose why we chose python
+>>>>>>> 2945e60ff7491f691f82f64eb82dc40acfff089b
```

Comment:
Why is this here??

Reply:
I was running into problems on my git and when I solved them this appeared in my changes

Reply:
it's still there. You should remove that.